azure-pipelines.yml

```
@@ -166,7 +166,7 @@ jobs:
     - bash: "python -m pip install -U pip setuptools --user"
       displayName: Upgrade pip and setuptools
 
-    - bash: "python -m pip install -U nbstripout==0.3.3 --user"
+    - bash: "python -m pip install -U "nbstripout>=0.3.5" --user"
```

Is this going to work with the added quotaton marks? Shouldn't this be:

```suggestion
    - bash: "python -m pip install -U nbstripout>=0.3.5 --user"
```
?

---

Might a slight change

---

But yes, double quotes there on inside and outside was an issue

---

> Might a slight change

Sorry?  :p

---

Yup, you were right the double quotes were an issue. I did still need them around the `"nbstripout>=0.3.5"` because `>` is a special bash symbol. But funnily enough I didn't need them around the full item. So I adjusted your suggestion, and then committed :)